File: 3st/Jaeyoung/특정 문자 제거하기.js
function solution(my_string, letter) {
    let answer = [];
    for ( var i = 0 ; i < my_string.length ; i++ ) {
        if ( my_string[i] == letter ) continue;
        else answer.push(my_string[i]);
    }
    
    return answer.join("");
}

File: 4st/Hani/programmer/ 특정 문자 제거하기.java
class Solution {
    public String solution(String my_string, String letter) {
        String answer = "";
        for (int i = 0; i < my_string.length(); i++) {
            char c = my_string.charAt(i);
            
            if (c != letter.charAt(0)) {
                answer += c;
            }
        }
        return answer;
    }
}